governance/settings/settings.php

```diff
@@ -108,11 +108,17 @@ public static function render() {
 		$post_types_available  = get_post_types();
 		$user_roles_available  = array_keys( wp_roles()->roles );
 		$governance_rules_json = GovernanceUtilities::get_governance_rules_json();
-		$governance_rules      = GovernanceUtilities::get_parsed_governance_rules();
-		$governance_error      = false;
-
-		if ( is_wp_error( $governance_rules ) ) {
-			$governance_error = $governance_rules->get_error_message();
+		if ( is_wp_error( $governance_rules_json ) ) {
+			$governance_error      = $governance_rules_json->get_error_message();
+			$governance_rules_json = '';
+		} else {
+			$governance_rules = GovernanceUtilities::get_parsed_governance_rules();
+			$governance_error = false;
```

Comment:
Super minor - I'd pull this line out before the `if` (line 110) to make it a little more obvious that `$governance_error` is scoped at root level and `false` is the default. PHP has weird variable hoisting that makes this work though, so it's fine if unchanged.